components/google_drive/sources/new-or-modified-comments/new-or-modified-comments.js

```diff
@@ -0,0 +1,185 @@
+// This source processes changes to any files in a user's Google Drive,
+// implementing strategy enumerated in the Push Notifications API docs:
+// https://developers.google.com/drive/api/v3/push and here:
+// https://developers.google.com/drive/api/v3/manage-changes
+//
+// This source has two interfaces:
+//
+// 1) The HTTP requests tied to changes in the user's Google Drive
+// 2) A timer that runs on regular intervals, renewing the notification channel as needed
+
+const { uuid } = require("uuidv4");
+const includes = require("lodash.includes");
```

Comment:
The usage of these per-method packages is [discouraged as per the Lodash docs](https://lodash.com/per-method-packages).

This should fix it:
```suggestion
const includes = require("lodash/includes");
```